### PR TITLE
use a custom query to create a more informative languoid.csv

### PR DESCRIPTION
**Pull request by xflr6**
- fix clld/glottolog#138

to test by creating only this download:
```python
import sys

from clld.scripts.util import parsed_args
from clld.interfaces import IDownload

sys.argv = ['spam', 'development.ini']

args = parsed_args(bootstrap=True, description='spam')

csv = dict(args.env['registry'].getUtilitiesFor(IDownload))['languoid.csv']

csv.create(args.env['request'])
```

### CODE REVIEW

**Comment by xflr6** [comment]
Not sure if we really want the `newick` column (at least Libreoffice seems to throw an error relazed to the maximal cell size).

**Comment by xflr6** [comment]
Still something wrong with the `iso639-3` column: let me take another look

**Comment by xflr6** [comment]
Fixed.

File: glottolog3/adapters.py [hunk 1]
```diff
@@ -39,15 +39,54 @@ def rec(self, ctx, req):
 class LanguoidCsvDump(CsvDump):
 
     def query(self, req):
-        return self.model.csv_query(session=req.db)
+        import sqlalchemy as sa
```
Comment (xrotwang) [inline glottolog3/adapters.py]: Can we avoid local imports? Makes the code a bit more difficult to inspect by pycharm or grep.

Comment (xflr6) [reply glottolog3/adapters.py]: Thanks. Pushed a new version that also omits the `newick` and `hid` columns.

**Pull request merged**